requirements.txt

```diff
@@ -19,4 +19,5 @@ pyqt5~=5.15.4
 qtpy~=1.11.2
 scikit-image~=0.18.3
 umap-learn~=0.5.1
-scikit-learn~=0.24.2
\ No newline at end of file
+scikit-learn~=0.24.2
+hdbscan~=0.8.27
```

Comment:
@lazigu I had to add hdbscan to the requirements because it didn't install automatically (I got an error when trying to run the hdscan code for the first time). I'm not sure if this is enough though?

Reply:
Hmm I thought that I messed up hdbscan clustering function and it was not returning what I expected as in kmeans function for example. But mayyybe it was related to installation. Strange that it was not working automatically.

Reply:
Oh so the function was also not working because you used the wrong function. Instead of generate_prediction_data() you should have just used fit_predict(measurement) just like in kmeans

Reply:
Ahhhh many thanks!! P.s. requirements.txt is a list for developers, while setup.cfg has a list what will be installed when using pip . So if hbdscan is there, strange that it’s not installed automatically (I can’t check now, I’m on a phone 🙈)

Reply:
It wasn't there before so thats probably why it didn't work but I haven't checked yet either and I'm trying to fix something else at the moment :D

Reply:
Ok no hurrying, I will look into it later!